rpcs3/Emu/RSX/gcm_enums.h

```diff
@@ -992,8 +992,35 @@ enum
 
 enum Method
 {
+	/*
```

Comment:
The original constants did things inaccurately. The correct way is in this PR, according to the envytools docs.